canonical/lxd-ui
# Move images into storage page, use custom iso label consistently

## Done

- move images page into storage page
- have tabs on storage page for pools, images and isos

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check storage section
    - check cached image tab under storages, launch an instance from there
    - check iso image tab under storages, launch an instance from there
